src/Mod/Spreadsheet/Gui/Command.cpp

```diff
@@ -564,8 +564,8 @@ CmdSpreadsheetAlignVCenter::CmdSpreadsheetAlignVCenter()
 {
     sAppModule = "Spreadsheet";
     sGroup = QT_TR_NOOP("Spreadsheet");
-    sMenuText = QT_TR_NOOP("&Vertically center-align");
-    sToolTipText = QT_TR_NOOP("Vertically center-align contents of selected cells");
+    sMenuText = QT_TR_NOOP("Align &Vertical Center");

```

Comment:
```suggestion
    sMenuText = QT_TR_NOOP("&Vertical Center Align");
```
Pending Discord discussion